packages/office-addin-sso/src/commands.ts

```diff
@@ -20,15 +20,15 @@ export async function configureSSO(manifestPath: string) {
     return;
   } else if (applicationDataConfigured(manifestPath)) {
     console.log(
-      chalk.yellow("Project has already been at least partially configured.  Can't setup new configuration completely.")
+      chalk.yellow("Project was already previously updated and will require manual steps when done.")
```

Comment:
What manual steps are needed? This isn't a very useful message for the user.

Consider something like. "Since the project has already been configured, please validate the configuration. See https://aka.ms/office-addin-sso for more information."  (You can create an aka.ms link to point to documentation. This allows it to be changed later if needed and not hardcoded.)

Reply:
There is an existing warning at the end of the script that says what needs to be done.  That is the one I missed when trying to test things and I thought the template was broken (it wasn't).  I wanted to add this at the beginning so that we wouldn't break an existing setup unless explicitly told to do so.

Reply:
I simplified the message and left the manual step info to the existing warning at the end of the script.